src/App.js

```diff
@@ -1,33 +1,30 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
+function useSemiPersistentState() {
+  const [todoList, setTodoList] = useState(JSON.parse(localStorage.getItem('savedTodoList')));
```

Comment:
Just a small tip. If you set todoList equal to an empty array, the app is going to work just in case the JSON doesn't parse correctly or there's nothing in local storage.
`JSON.parse(localStorage.getItem("savedTodoList")) || []`

Reply:
Hello Xeniya, 
Thanks for the feedback.